tests/test_recipient_validation.py

```diff
@@ -298,7 +298,7 @@ def test_normalise_phone_number_raises_if_unparseable_characters(phone_number):
 def test_get_international_info_raises(phone_number):
     with pytest.raises(InvalidPhoneError) as error:
         get_international_phone_info(phone_number)
-    assert str(error.value) == "Not a valid country prefix"
+    assert str(error.value) == "Country code not found - double check this mobile number"
```

Comment:
```suggestion
    assert str(error.value) == "Country code not found -  double check the number you entered"
```

Worth making the instruction consistent with the message we use for invalid UK numbers?